### PR TITLE
Add attribute delimiter

### PR DESCRIPTION
**Pull request by romkatsu**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #19

### CODE REVIEW

**Review by rustamwin — APPROVED** [review]

**Review by vjik — CHANGES_REQUESTED** [review]
Seems this implementation will not be convenient. If we don't use dot as delimiter, then need always run method `withAttributeDelimiter`. Also delimiter is internal data of model, this is don't sholud be set external. I see two variants:

1) Add protected method `getDeilimitter(): ?string` (delimiter also can be null). And by default delimiter make is `null`. I like this variant.

2) Remove support delimiter in `RequestModel` and create yet one abstract class `RequestByPathModel` with method `getDeilimitter(): string` (here delimiter can be only string).

**Comment by romkatsu** [comment]
> Seems this implementation will not be convenient. If we don't use dot as delimiter, then need always run method `withAttributeDelimiter`. Also delimiter is internal data of model, this is don't sholud be set external. I see two variants:
> 
>     1. Add protected method `getDeilimiter(): ?string` (delimiter also can be null). And by default delimiter make is `null`. I like this variant.
> 
>     2. Remove support delimiter in `RequestModel` and create yet one abstract class `RequestByPathModel` with method `getDeilimitter(): string` (here delimiter can be only string).

I agree. The public method is not needed. Left only the property.

File: src/RequestModel.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 abstract class RequestModel implements RequestModelInterface
 {
     private array $requestData = [];
+    protected string $attributeDelimiter = '.';
```
Comment (vjik) [inline src/RequestModel.php:12]: How about make this nullable and by default set `null` ?

Comment (romkatsu) [reply src/RequestModel.php:12]: It seems to me so more explicitly that a dot will be used

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**